client/public/locales/en/translation.json

```diff
@@ -0,0 +1,28 @@
+{
+  "planYourTrip": "Plan your trip",
+  "emailAddress": "Email address",
+  "yourName": "Your name",
+  "password": "Password",
+  "sendInstructions": "Send instructions",
+  "checkEmail": "Check your email. We have sent you some information.",
+  "goToLogin": "Go to login",
+  "welcomeToUseeSvit": "Welcome to UseeSvit.",
+  "somethingWentWrong": "Something went wrong",
+  "logIn": {
+    "title": "Log In",
+    "btn": "LOGIN"
+  },
+  "signUp": {
+    "title": "Sign Up",
+    "btn": "SIGN UP"
+  },
+  "forgotPassword": {
+    "title": "Forgot your password?",
+    "btn": "Forgot password"
+  },
+  "profileMenu": {
+    "profile": "Profile",
+    "myAccount": "My account",
+    "logOut": "Logout"
+  }
+}
```

Comment:
<img width="134" alt="Screenshot 2023-06-28 at 10 44 49" src="https://github.com/mykhalenych/someName/assets/53228259/5a10e69a-9fb1-4d57-8620-f15d5a23f0fb">
add empty line in the end of the file